source/localizable/smart_cart/webhook.html.md.erb

```diff
@@ -113,6 +113,67 @@ Name           | Type   | Value | Description
 
 <%= partial 'localizable/smart_cart/order_object' %>
 
+## Test requests (sandbox)
+
+For [demo orders](/smart_cart/orders_api/#test-requests-sandbox), separate endpoints are available to trigger different
+kinds of webhook requests for testing. In the following examples, `:code` should be replaced with one of the following values:
+
+- `DEMO-OPEN`
+- `DEMO-ACCEPTED`
+- `DEMO-REJECTED`
+- `DEMO-INVOICE`
+- `DEMO-INVOICE39A`
+
+An order creation or order update event should be received at the webhook url of the shop, depending on the request action.
+Sending any of the following requests should request in a successful response (`200 OK`):
+
+<pre class="terminal">
+  {
+    "success": true
+  }
+</pre>
+
+
+> Note: Demo webhook requests are not executed instantly, so they may take from a few seconds up to a few minutes to be processed and sent.
+> After getting a successful response (200 OK), you would just have to wait a bit, before trying another one.
+> All requests will be sent to the live Webhook URL that is already set-up in the Merchants panel for your shop.
+
+#### New order
+
+This sends a Webhook request payload just like if the order is new, with `"event_type":
+“new_order”`.
```

Comment:
Minor, fix word's backquotes to normal double quotes: `“new_order”` >>>> `"new_order"`